src/components/AlertCard/index.tsx

```diff
@@ -1,78 +1,160 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
-import React, { HTMLProps } from 'react'
-
+import React from 'react'
 import classNames from 'classnames'
-
-import { CloseButton, RightArrow, WarningIcon } from '@/assets/images'
+import { RightArrow } from '@/assets/images'
 import Button from '@/components/Button'
-
 import styles from './index.module.scss'
+import { ReactComponent as CloseButton } from '@/assets/images/CloseButton.svg'
 
-type Variants = 'warning' | 'danger'
+type Tone =
+  | 'error'
+  | 'warning'
+  | 'success'
+  | 'information'
+  | 'neutral'
+  | 'brand'
+type Size = 'small' | 'large'
+type Layout = 'horizontal' | 'vertical'
 
-export interface InfoAlertProps extends HTMLProps<HTMLDivElement> {
+export interface InfoAlertProps {
   title: string
   message: string
```

Comment:
We need to change this `message` to accept a ReactNode otherwise it is not possible to style the text inside it. For instance, if we want to add parts of the string to be bold, italic, etc.

```suggestion
  message: React.ReactNode
```

I added this on this PR due to the ThirdParty downloader alert I need to add to the client, but you can do it on this PR and then I can close mine:
https://github.com/HyperPlay-Gaming/hyperplay-ui/pull/497/files